db/seeds.rb

```diff
@@ -20,9 +20,20 @@
 #   puts "Added #{label_name}!"
 # end
 
+# Past trips seed 1.0
+User.destroy_all
+user = User.create!([
```

Comment:
I added the first and last name to user.
New users need avatars. to do so, you need to user User.new then attach it then save (see reference in rentreks seed.) I added it to db/seed_src/(REF)_rentrek_seed.rb as reference.